x/ethbridge/handler.go

```diff
@@ -67,17 +70,55 @@ func handleMsgCreateEthBridgeClaim(ctx sdk.Context, cdc *codec.Codec,
 	return sdk.Result{Events: ctx.EventManager().Events()}
 }
 
-func processSuccessfulClaim(ctx sdk.Context, bankKeeper bank.Keeper, claim string) sdk.Error {
+func processSuccessfulClaim(ctx sdk.Context, supplyKeeper supply.Keeper, claim string) sdk.Error {
```

Comment:
This function should prob live on a ethbridge `Keeper`

Reply:
will refactor to use expected_keepers and to create a bridge keeper in follow up PR